<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayMessageClass</class>
 <widget class="QMainWindow" name="DisplayMessageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1663</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplayMessage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>512</width>
      <height>375</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>1082</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>511</width>
      <height>292</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="openCloudButton">
         <property name="text">
          <string>打开点云</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="openTableButton">
       <property name="text">
        <string>打开数据表格</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="selectButton">
         <property name="text">
          <string>筛选数据</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>格式为yyyy-mm-dd hh-mm-ss</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lowEdit">
           <property name="text">
            <string>2018-01-01 00-00-00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>至</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="upEdit">
           <property name="text">
            <string>2018-01-02 00-00-00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>点云文件名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cloudFileDirEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>表格文件名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tableFileDirEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>圆心：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="centerEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>平面法矢：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="planeNormalEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>半径：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="radiusEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="selectCenterButton">
           <property name="text">
            <string>选择圆心</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="displayCenterButton">
           <property name="text">
            <string>显示圆心</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1663</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="displaymessage.qrc"/>
 </resources>
 <connections/>
</ui>
